.github/workflows/end2end.yml

```diff
@@ -34,6 +34,12 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Set environment variables for generate_contract_input.py (Nova URL and commit)
+        id: parse-comment
+        run:
+          echo "::set-env name=NOVA_URL::https://github.com/lurk-lab/Nova"
```

Comment:
@samuelburnham , as we discussed - it would be nice to make specifying `NOVA_URL` and `NOVA_COMMIT` via `!` `test` comment. In case if they are specified in comment, it is necessary to take them instead of extracting from `rust-reference-info.txt`. As you are able to carefully test this behaviour it is better to have a separate PR from you